scripts/run_kickstart_tests.sh

```diff
@@ -176,8 +176,10 @@ if [[ $? -ne 0 ]]; then
 fi
 ISO_OS_NAME=$(echo "${output}" | grep 'NAME=')
 ISO_OS_NAME="${ISO_OS_NAME##NAME=}"
+export KSTEST_OS_NAME="${ISO_OS_NAME}"
```

Comment:
This should not be needed as in ks.in files we use @KSTEST_OS_NAME@ and @KSTEST_OS_VERSION@ macros as in https://github.com/rhinstaller/kickstart-tests/blob/208da7737a2662663634855cfa005ae8ccdd8b77/repo-install.ks.in#L20 (they are substituted with the values 5 lines below). (for the reference, documented here: https://github.com/rhinstaller/kickstart-tests/blob/master/README.rst#chapter-4-environment-variables)